docker/bitcoind/Dockerfile

```diff
@@ -1,19 +1,36 @@
-# Use the official Ubuntu 22.04 base image
+# Use the official Ubuntu 24.04 base image
 FROM ubuntu:24.04
 
 WORKDIR /app
 
-# Install necessary dependencies and add PPA
+# Install necessary dependencies
 RUN apt-get update && \
-    apt-get install -y software-properties-common && \
-    add-apt-repository ppa:luke-jr/bitcoincore && \
-    apt-get update && \
-    apt-get install -y \
-    bitcoind \
-    curl && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
+    apt-get install -y software-properties-common build-essential libtool autotools-dev automake pkg-config bsdmainutils python3 && \
+    apt-get install -y libevent-dev libboost-dev libsqlite3-dev && \
+    apt-get install -y libssl-dev libboost-all-dev git
```

Comment:
I think it is better to replace three `apt-get` command with single one if we are concerned about number of layers in docker image.

Reply:
I think this will create only 1 layer.

Reply:
Anyway addressed this.